### PR TITLE
Add support guidelines viewable on GitHub w/ SUPPORT.md

### LINKED ISSUE #166: Use SUPPORT
Should we? For tidyverse packages, we have obvious places to link to, e.g. contributing guidelines, community.rstudio.com, [non-code contributions](https://www.tidyverse.org/contribute/).

https://help.github.com/articles/adding-support-resources-to-your-project/

### PR DESCRIPTION
**Pull request by batpigandme**
* Closes #166.

### CODE REVIEW

File: R/tidyverse.R [hunk 1]
```diff
@@ -23,9 +23,10 @@
 #'
 #' * `use_tidy_issue_template()`: creates a standard tidyverse issue template.
 #'
-#' @md
+#' * `use_tidy_support`: creates support resources document for GitHub repo
+#'    using SUPPORT.md.
+#'
 #' @name tidyverse
-NULL
```
Comment (jennybc) [inline R/tidyverse.R]: This `NULL` line needs to stay. This is the area of the merge conflict as well. Do you want to try to straighten it out in your branch/PR? If not, I will resolve.

Comment (batpigandme) [reply R/tidyverse.R]: Thanks, Jenny. I _think_ I resolved them!

File: inst/templates/SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Getting help with {{{ Package }}}
+
+Thanks for using {{{ package }}}. Before filing an issue, please take a minute
+to look at these materials so that you can get the help you need as quickly as
+possible.
+
+The [Get help!](https://www.tidyverse.org/help/) section of the tidyverse site
```
Comment (hadley) [inline inst/templates/SUPPORT.md]: Can we make a little more fun? Something like:

> Start by making a reprex. If you don't know what a reprex is, STOP WHAT YOU ARE DOING:
> you are in for a treat

File: inst/templates/SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+is a good place to look for general guidance, including how to make a minimal
+**repr**oducible **ex**ample (AKA a [reprex](http://reprex.tidyverse.org/)).
+
+Armed with your reprex, your next step is to figure out [where to ask](https://www.tidyverse.org/help/#where-to-ask) your question. We use GitHub 
+Issues for bugs, and feature requests.
+
+For questions and help with using {{{ package }}}, 
+[community.rstudio.com](https://community.rstudio.com/), and 
+[StackOverflow.com](https://stackoverflow.com/) are better places to start. Not
```
Comment (hadley) [inline inst/templates/SUPPORT.md]: Just "StackOverflow", I think

File: inst/templates/SUPPORT.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+[StackOverflow.com](https://stackoverflow.com/) are better places to start. Not 
+only are there more people there to answer questions, it's also a good way to 
+figure out if your problem _is_ an issue, in which case you can easily return 
+here and report it. 
+
+Before opening a new issue, be sure to [search issues and pull requests](https://help.github.com/articles/searching-issues-and-pull-requests/) 
+to make sure the bug hasn't been reported and/or already fixed in the 
+development version.
+
+If you _are_ in the right place, and need to file an issue, please review the
```
Comment (hadley) [inline inst/templates/SUPPORT.md]: Can we inline this advice?

File: inst/templates/SUPPORT.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Thanks for using {{{ package }}}. Before filing an issue, please take a minute
+to look at these materials so that you can get the help you need as quickly as
+possible.
+
+The [Get help!](https://www.tidyverse.org/help/) section of the tidyverse site
+is a good place to look for general guidance, including how to make a minimal
+**repr**oducible **ex**ample (AKA a [reprex](http://reprex.tidyverse.org/)).
+
+Armed with your reprex, your next step is to figure out [where to ask](https://www.tidyverse.org/help/#where-to-ask) your question. We use GitHub
```
Comment (hadley) [inline inst/templates/SUPPORT.md]: Maybe make into a bulleted list?

* If it's a question:
* If it's a bug:
* If you're not sure:

**Pull request merged**